package.json

```diff
@@ -3,9 +3,13 @@
   "version": "0.0.1",
   "description": "Web API Confluence Dashboard",
   "scripts": {
-    "coverage": "karma start ./config/karma.coverage.conf.js",
-    "test": "karma start ./config/karma.all.conf.js",
-    "serve": "zsh scripts/serve.sh"
+    "coverage": "npm run coverageNode && npm run coverageWeb && istanbul-combine -d .coverage -p summary -r lcov -r html -r json -b . .node_coverage/coverage.json .web_coverage/*/coverage*.json && istanbul check-coverage --config config/istanbul.yml \".coverage/**/coverage*.json\"",
```

Comment:
Again, I think we're stuck with this because of the requirement for valid JSON.  We could punt to a shell script. IMHO that adds unnecessary indirection. @arobins WDYT?